opencog/nlp/learn/README

```diff
@@ -180,7 +180,7 @@ So, set up the text-ingestion pipeline:
    for parsing, counts the returned word-pairs, and stores them in
    the database.
 
-8a) Verify that the above resulted in data sent to the SQL database.
+7a) Verify that the above resulted in data sent to the SQL database.
```

Comment:
Once again -- if you are renumbering everything anyway, you may as well get rid of the a,b,c numbering too.  The only reason that there is an a,b,c is because I did not want to waste time renumbering everything. its pointless to preserve this hack.